grid/src/Cabana_Grid_SparseLocalGrid_impl.hpp

```diff
@@ -276,7 +276,9 @@ auto LocalGrid<SparseMesh<Scalar, NumSpaceDim>>::sharedTileIndexSpaceImpl(
             min[d] = ( owned_cell_space.max( d ) - halo_width ) >>
                      cellBitsPerTileDim;
         else
-            throw std::runtime_error( "Neighbor offset must be 1, 0, or -1" );
+            throw std::runtime_error(
+                "Cabana::Grid::Experimental::sharedTileIndexSpaceImpl "
```

Comment:
Let's remove the `Impl` from all these (the top level function is `sharedTileIndexSpace`)